pkg/app/pipedv1/plugin/kubernetes/deployment/plugin.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/pipe-cd/pipecd/pkg/app/pipedv1/plugin/kubernetes/toolregistry"
 	config "github.com/pipe-cd/pipecd/pkg/configv1"
 	"github.com/pipe-cd/pipecd/pkg/plugin/sdk"
+	"go.uber.org/zap"
```

Comment:
Please move this import to before the imports like `github.com/pipe-cd/pipecd/**`

Reply:
@Warashi 
Sorry, fixed on [7d8e4a2](https://github.com/pipe-cd/pipecd/pull/5657/commits/7d8e4a26ac7a6c862ef987957c0bb190c7350ba6) and fixed my IDE's import setting 🙏